lib/fastlane/plugin/firebase_app_distribution/client/aab_info.rb

```diff
@@ -0,0 +1,40 @@
+class AabInfo
+  # AAB states
+  class AabState
+    UNSPECIFIED = 'AAB_STATE_UNSPECIFIED'
+    PLAY_ACCOUNT_NOT_LINKED = 'PLAY_ACCOUNT_NOT_LINKED'
+    NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT = 'NO_APP_WITH_GIVEN_BUNDLE_ID_IN_PLAY_ACCOUNT'
+    APP_NOT_PUBLISHED = 'APP_NOT_PUBLISHED'
+    PLAY_IAS_TERMS_NOT_ACCEPTED = 'PLAY_IAS_TERMS_NOT_ACCEPTED'
+    ACTIVE = 'ACTIVE'
+    UNAVAILABLE = 'AAB_STATE_UNAVAILABLE'
+  end
+
+  def initialize(response)
+    @response = response || {}
+  end
+
+  def integration_state
+    @response[:integrationState]
+  end
+
+  def test_certificate
+    @response[:testCertificate] || {}
+  end
+
+  def md5_certificate_hash
+    test_certificate[:hashMd5]
+  end
+
+  def sha1_certificate_hash
+    test_certificate[:hashSha1]
+  end
+
+  def sha256_certificate_hash
+    test_certificate[:hashSha256]
+  end
+
+  def certs_provided?
+    !!md5_certificate_hash && !!sha1_certificate_hash && !!sha256_certificate_hash
```

Comment:
I believe this has a slightly different meaning than what the prior code was...this is _only_ a nilness check on all three, so it will treat empty hashes as `provided`

Reply:
Good call, updated to the previous double protection, just negated.